var/spack/repos/builtin/packages/whizard/package.py

```diff
@@ -100,6 +102,8 @@ def setup_build_environment(self, env):
     def configure_args(self):
         spec = self.spec
         args = [
+            "TIRPC_CFLAGS=-I%s/include/tirpc" % spec["libtirpc"].prefix,
```

Comment:
FYI. You could "simplify" with (assuming `tirpc` is also a directory):

```suggestion
            "TIRPC_CFLAGS=-I%s" % spec["libtirpc"].prefix.include.tirpc,
```

Reply:
Thanks for the suggestions @tldahlgren - yes, that does work in this case.